dashboards/resources.libsonnet

```diff
@@ -2,53 +2,153 @@ local g = import 'grafana-builder/grafana.libsonnet';
 
 {
   grafanaDashboards+:: {
+    'k8s-resources-multicluster.json':
+      local tableStyles = {
+        cluster: { //TODO: Does this need to be "%(clusterLabel)s" ??
+          alias: 'Cluster',
+          link: '%(prefix)s/d/%(uid)s/k8s-resources-cluster?var-datasource=$datasource&var-cluster=$__cell' % { prefix: $._config.grafanaPrefix, uid: std.md5('k8s-resources-cluster.json') },
+        },
+      };
+      
+      g.dashboard(
+        '%(grafanaDashboardNamePrefix)s  Compute Resources /  Multi-Cluster' % $._config,
+        uid=($._config.grafanaDashboardIDs['k8s-resources-multicluster.json']),
+      ).addRow(
+        (g.row('Headlines') +
+         {
+           height: '100px',
+           showTitle: false,
+         })
+         .addPanel(
+           g.panel('CPU Utilisation') +
+           g.statPanel('1 - avg(rate(node_cpu_seconds_total{mode="idle", %(clusterLabel)s=~".*"}[1m]))' % $._config)
```

Comment:
`=~".*"` labels match anything (including empty), so this isn't doing anything right now. Do you mean all of these to be `%(clusterLabel)s!=""`?

Reply:
doh!! missed this. I think we can actually omit this label match clause entirely on these top-multi-cluster dashboards. In all but the single-stat, we're grouping by cluster, I don't think we even need `cluster!=""`

Reply:
I think it would be more robust to do `cluster!=""` in case there is any data that does not belong to a cluster. I don't feel super strongly about that though.